core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/dot-uve-toolbar/components/dot-uve-workflow-actions/dot-uve-workflow-actions.component.spec.ts

```diff
@@ -90,7 +90,7 @@ const uveStoreMock = {
     pageAPIResponse: signal(MOCK_RESPONSE_VTL),
     workflowActions: signal([]),
     workflowLoading: signal(false),
-    canEditPage: signal(true),
+    $canEditPage: signal(true),
```

Comment:
is this needed? I know we should use it like that but it breaks the pattern, we might need to change them all or leave it as it was